lib/microservice_precompiler/builder.rb

```diff
@@ -156,9 +156,10 @@ def sprockets_env
     def minify(asset, format)
       asset = asset.to_s
       # Minify JS
-      return Uglifier.compile(asset) if format.eql?(".js")
+      return Uglifier.compile(asset) if format.eql?("js")
+      # CSS COMPRESSOR NOT NEEDED AS IT IS ALREADY COMPACT
       # Minify CSS
-      return YUI::CssCompressor.new.compress(asset) if format.eql?(".css")
+      #return YUI::CssCompressor.new.compress(asset) if format.eql?(".css")
```

Comment:
@Amay22 Sprockets does not automatically minimize the file, it just compiles the requires files and translates scss to css. We still need this step to minimize the file.